gax-java/gax-httpjson/src/test/java/com/google/api/gax/httpjson/HttpJsonDirectCallableTest.java

```diff
@@ -72,7 +74,19 @@ public class HttpJsonDirectCallableTest {
                         serializer.putPathParam(fields, "name", request.getName());
                         return fields;
                       })
-                  .setAdditionalPaths("/fake/v1/name/{name=john/*}")
+                  .setAdditionalPathsExtractorMap(
+                      ImmutableMap
+                          .<PathTemplate, FieldsExtractor<Field, Map<String, String>>>builder()
+                          .put(
+                              PathTemplate.create("/fake/v1/name/{name=john/*}"),
+                              request -> {
+                                Map<String, String> fields = new HashMap<>();
+                                ProtoRestSerializer<Field> serializer =
+                                    ProtoRestSerializer.create();
+                                serializer.putPathParam(fields, "name", request.getName());
+                                return fields;
+                              })
+                          .build())
```

Comment:
The test cases use `john/imTheBestField` so they match against the additionalPathExtractor()